webapp/components/test-file-results-table.js

```diff
@@ -195,7 +195,7 @@ class TestFileResultsTable extends WPTFlags(Pluralizer(AmendMetadataMixin(WPTCol
     </template>
   </tbody>
 </table>
-<wpt-amend-metadata id="amend" selected-metadata="{{selectedMetadata}}" path="[[path]]"></wpt-amend-metadata>
+<wpt-amend-metadata id="amend" selected-metadata="[[selectedMetadata]]" path="[[path]]"></wpt-amend-metadata>
```

Comment:
we still need to listen to the change in `selectedMetadata` in test-file-results-table.js, because it is used to clear the selected cells UI when the triage UI is dismissed through `clearSelectedCells`. I will post a recorded video clips to demonstrate that

Reply:
Good catch! It's actually the method in `wpt-results.js` that clears the cells, but I changed it here for consistency.